src/igvfd/tests/test_schema_sample_ontology_term.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+
+def test_sample_ontology_term_term_id_regex(sample_ontology_term_1, testapp):
+    res = testapp.patch_json(
+        sample_ontology_term_1['@id'],
+        {'term_id': 'ABC:12345'},
+        expect_errors=True)
+    assert(res.status_code == 422)
```

Comment:
Should we have a test that checks that when you put in some known cell that you are getting reasonable slims upon posting?